benchmarks/benchmarks/stats.py

```diff
@@ -20,3 +22,60 @@ def time_anderson_ksamp(self):
         with warnings.catch_warnings():
             warnings.simplefilter('ignore', UserWarning)
             anderson_ksamp(self.rand)
+
+def random(size):
+    return rand(*size)
+
+class CorrelationFunctions(Benchmark):
+    param_names = ['alternative']
+    params = [
+        ['two-sided', 'less', 'greater']
+    ]
+
+    def setup(self, mode):
+        a = random([2,2])*10
+        self.a = a
+
+    def time_fisher_exact(self, alternative):
+        if alternative == 'two-sided':
+            oddsratio, pvalue = fisher_exact(self.a, alternative='two-sided')
+        elif alternative == 'less':
+            oddsratio, pvalue = fisher_exact(self.a, alternative='less')
+        else:
+            oddsratio, pvalue = fisher_exact(self.a, alternative='greater')
+
+class InferentialStats(Benchmark):
+
+    def setup(self):
+        np.random.seed(12345678)
+        rvs1 = stats.norm.rvs(loc=5,scale=10,size=500)
+        rvs2 = stats.norm.rvs(loc=5,scale=10,size=500)
+        rvs3 = stats.norm.rvs(loc=5, scale=20, size=500)
+        rvs4 = stats.norm.rvs(loc=5, scale=20, size=100)
+        rvs5 = stats.norm.rvs(loc=8, scale=20, size=100)
+        self.a = rvs1
+        self.b = rvs2
+        self.c = rvs3
+        self.d = rvs4
+        self.e = rvs5
+
+    def time_ttest_ind_same_mean(self):
+        # test with sample with identical means:
+        stats.ttest_ind(self.a, self.b)
+        stats.ttest_ind(self.a, self.b, equal_var=False)
+
+    def time_ttest_ind_diff_var(self):
+        # test with sample with unequal variances:
+        stats.ttest_ind(self.a, self.c)
+        stats.ttest_ind(self.a, self.c, equal_var=False)
+
+    def time_ttest_ind_diff_size(self):
+        # test with sample with different sizes
```

Comment:
good idea to test with different sizes. i would make the size difference larger though - I expect 500/100 to behave similarly to 500/500. Maybe make it 500/20 or some such ratio?